Dockerfile

```
@@ -1,5 +1,5 @@
 # This ARG has to be at the top, otherwise the docker daemon does not known what to do with FROM ${RUNTIME_IMAGE}
-ARG RUNTIME_IMAGE=alpine:3.15
+ARG RUNTIME_IMAGE=alpine:3.16
```

```suggestion
ARG RUNTIME_IMAGE=alpine:3.16.2
```

Better IMO, since this tag could be considered immutable. But that will add a potential maintenance burden. Is it possible to get Dependabot (or similar) to suggest upgrades?

The main problem in this project is the infrequent releases....